SDK/bundles/org.eclipse.embedcdt.debug.gdbjtag.core/src/org/eclipse/embedcdt/debug/gdbjtag/core/dsf/GnuMcuFinalLaunchSequence.java

```diff
@@ -389,6 +393,43 @@ public void stepGnuMcuStart(RequestMonitor rm) {
 		queueCommands(commandsList, rm);
 	}
 
+	@Execute
+	public void stepAddTargetDescription(final RequestMonitor rm) {
```

Comment:
Add a function comment so that all can understand it's setting register description file.

Reply:
Comment added